engines/director/lingo/lingo-the.cpp

```diff
@@ -1105,7 +1105,12 @@ void Lingo::setTheEntity(int entity, Datum &id, int field, Datum &d) {
 		break;
 	case kTheTimeoutLapsed:
 		// timeOutLapsed can be set in D4, but can't in D3. see D3.1 interactivity manual p312 and D4 dictionary p296.
-		setTheEntitySTUB(kTheTimeoutLapsed);
+		if (g_director->getVersion() >= 400 && d.type == INT) {
+			g_director->_tickBaseline = g_director->getMacTicks() - d.asInt();
+		}
+		if (d.type != INT) {
+			warning("Lingo::setTheEntity() : Wrong DatumType %d for setting of Lingo Property timeOutLapsed", d.type);
```

Comment:
I'll merge this, but something tells me that Lingo will be fine if the type is FLOAT too.